tox.ini

```diff
@@ -8,6 +8,7 @@ deps =
     pytest
     virtualenv
     py3: coveralls
+    backports.tempfile
```

Comment:
Basically backports Python [TemporaryDirectory context manager](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) to Python2.
It clearly simply things up as it avoids having to reinvent the wheel while still supporting Python2. This dep is just a test dep and doesn't impact the rests of the project.